.yarnrc

```diff
@@ -1,6 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
-
-
-lastUpdateCheck 1605737438433
-yarn-path ".yarn/releases/yarn-1.22.10.js"
```

Comment:
@Rugvip is this a command pattern for Yarn? I actually had to delete this file before I could run `yarn set version stable` without getting an error about stable not being found. If this is common then it might be something we add to the migration doc: https://backstage.io/docs/tutorials/yarn-migration